README.md

```diff
@@ -213,12 +143,7 @@ Example:
 - GitHub: [@githubhandle](https://github.com/githubhandle)
 - Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
Great job @AhmadiGu 🤝

- [ ] Kindly make sure the social handle links for this author work properly as intended. This will make your readme file more professional.